apps/api/src/server/applications/application/representations/publishable/__tests__/publishable-representations.test.js

```diff
@@ -46,41 +46,89 @@ const representations = [
 describe('Get Publishable Representations', () => {
 	afterEach(() => jest.clearAllMocks());
 
-	it('returns publishable representations for given case id', async () => {
+	describe('for given case id', () => {
 		databaseConnector.representation.findMany.mockResolvedValue(representations);
 
-		const response = await request.get('/applications/1/representations/publishable');
+		describe('when representations have been published before', () => {
+			it('returns publishable representations with .previouslyPublished true', async () => {
+				databaseConnector.representation.count.mockResolvedValue(2);
```

Comment:
would it makes sense for this to return `1`, as the `representations` list contains one PUBLISHED and one VALID?

```suggestion
				databaseConnector.representation.count.mockResolvedValue(1);
```

Reply:
That makes more sense - thank you!